scss/button/_theme.scss

```diff
@@ -76,13 +76,19 @@
         // Focused state
         .k-button:focus,
         .k-button.k-state-focused {
-            @include appearance( focused-button );
+            box-shadow: inset 0 0 0 2px rgba(0, 0, 0, .13);
```

Comment:
I was thinking on that... However, we have two different sets of box shadow: one for "standalone" buttons and another for other types.

I was thinking perhaps a "generic" `$box-shadow` variable might do the trick, same as `$border-radius`...